build/tools/amebasmart/bootloader/target_loader.map

```diff
@@ -117,297 +117,297 @@
+3000805c T CLK_SWITCH_XTAL
+300080a4 T DDR_PHY_ChipInfo_ddrtype
+300080c0 T DDR_PHY_StructInit
+300080c8 T DDR_PHY_DLL_CLK_DIV
+300084b4 T DDR_PHY_CRT_Init
+30008584 T DDR_PHY_R240_ZQ_CAL
+300087e0 T DDR_PHY_ZQ_SET_SEL
+30008888 T DDR_PHY_DELAY_TAP_SET
+300088f8 T DDR_PHY_READ_CTRL
+300089b0 T DDR_PHY_READ_LEVELING
+30008a98 T DDR_PHY_Init
+30008b68 T DDR_PHY_CAL
+30008b6c T DDR_PHY_AutoGating
+30008b9c T flash_init_userdef
+30008bfc T nand_init_userdef
+30008c70 T flash_layout_init
+30008e34 T NAND_CHECK_IS_BAD_BLOCK
+30008ee4 T Nand_Get_NandAddr
+30008f30 T NandImgCopy
+300090a4 t OTP_Read8.part.0
+30009184 t OTP_ProgramMarginRead8.part.0
+30009268 T OTP_Read8
+3000927c T OTP_Write8
+30009428 T EFUSERead8
+30009440 T EFUSE_PMAP_READ8
+30009458 T PLL_NP_ClkSet
+300094d8 T PLL_AP_ClkSet
+30009518 T PLL_AP
+30009578 T PSRAM_INFO_Update
+300096bc T PSRAM_PHY_StructInit
+300096dc T PSRAM_PHY_Init
+30009744 T PSRAM_CTRL_Init
+30009888 T PSRAM_REG_Write
+30009910 T PSRAM_APM_DEVIC_Init
+30009968 T PSRAM_WB_REG_Write
+300099e8 T PSRAM_WB_DEVIC_Init
+30009a24 T PSRAM_calibration
+30009cac T PSRAM_AutoGating
+30009cf8 T BOOT_Reason
+30009d08 T CRYPTO_SetSecurityModeAD
+30009eb0 T rtl_crypto_hmac_sha2_init
+3000a198 T CPU_ClkGet
+3000a238 T RSIP_IV_Set
+3000a254 T RSIP_OTF_Cmd
+3000a26c T RSIP_OTF_Enable
+3000a2e8 T RSIP_MMU_Config
+3000a328 T RSIP_MMU_Cmd
+3000a350 T RSIP_MMU_Cache_Clean
+3000a360 T NAND_RxData
+3000a498 T NAND_Page_Read
+3000a4c8 T irq_register
+3000a540 T SYSCFG_BootFromNor
+3000a588 T SYSCFG_RLVersion
+3000a5ac T ddr_init_index
+3000a600 T rxi316_perf_tune
+3000aeb8 T rxi316_dram_init
+3000b8f4 T rxi316_ftend_init
+3000b934 T rxi316_DynSre_init
+3000b98c T ddr_init
+3000b9b4 T SWR_MEM
+3000ba94 T SWR_MEM_Manual
+3000bab4 T SWR_MEM_Mode_Set
+3000bb54 T SWR_AUDIO
+3000bbe4 T SWR_AUDIO_Manual
+3000bc04 T SWR_BST_MODE_Set
+3000bc24 T SWR_Calib_DCore
+3000bc54 T SWR_Calib_MEM
+3000bda4 T SWR_Calib_AUD
+3000bec8 T __aeabi_uldivmod
+3000bef8 T __udivmoddi4
+3000c1a0 W __aeabi_idiv0
+3000c1a0 W __aeabi_ldiv0
+3000c1a8 t ____wrap_strlen_veneer
+3000c1b8 t __SBOOT_GetMdType_veneer
+3000c1c8 t __SBOOT_Validate_Algorithm_veneer
+3000c1d8 t __SBOOT_Validate_Signature_veneer
+3000c1e8 t ____wrap_memset_veneer
+3000c1f8 t ____wrap_memcpy_veneer
+3000c208 t __io_assert_failed_veneer
+3000c218 t __SBOOT_Validate_ImgHash_veneer
+3000c228 t __DiagPrintf_veneer
+3000c238 t __SBOOT_Validate_PubKey_veneer
+3000c248 t ____wrap_memcmp_veneer
+3000d1c4 T Memory_Info
+3000d278 T Flash_AVL
+3000d37c T Flash_ReadMode
+3000d37e T Flash_Speed
+3000d380 T NAND_AVL
+3000d420 t __FUNCTION__.0
+3000d430 t __FUNCTION__.1
+3000d5a8 t __FUNCTION__.1
+3000d64c t __FUNCTION__.0
+3000d660 t __FUNCTION__.1
+3000d670 t __FUNCTION__.2
+3000d68c t __FUNCTION__.3
+3000d748 t __FUNCTION__.1
+3000d7c0 t __FUNCTION__.0
+3000d7d0 T km4_bootloader_rev
+3000d808 D Boot_Agg_En
+3000d808 D __ram_image1_data_start__
+3000d809 D Boot_MemSwr_Only
+3000d80a D Boot_AP_Enbale
```

Comment:
Will you modify typo? Enable

Reply:
This is not request, just asking.

Reply:
Dear @sunghan-chang,
We will not modify this typo in this PR, we will modify it in the future PR for bootloader. Is that okay?

Reply:
Sure